google-cloud-spanner/test/google/cloud/spanner/convert/to_query_params_test.rb

```diff
@@ -254,6 +254,17 @@
                                              Google::Spanner::V1::Type.new(code: :NUMERIC)] })
   end
 
+  it "converts a Hash to json value " do
+    value = { name: "ABC", open: true, rating: 10 }
+    combined_params = Google::Cloud::Spanner::Convert.to_query_params(
```

Comment:
combined_params => query_params?

Reply:
In other tests same naming convention followed for query params.